modules/images/webp-uploads/load.php

```diff
@@ -742,3 +742,33 @@ function webp_uploads_get_image_sizes_additional_mime_type_support() {
 
 	return $allowed_sizes;
 }
+
+/**
+ * Updates the quality of images used in the wp editor.
+ * For WP version less than 6.2 returns 82,
+ * Otherwise returns default quality.
+ *
+ * @since n.e.x.t
+ *
+ * @param int    $quality   Quality level between 1 (low) and 100 (high).
+ * @param string $mime_type Image mime type.
+ * @return int The updated quality for mime types.
+ */
+function webp_uploads_set_wp_image_editor_quality( $quality, $mime_type ) {
+	global $wp_version;
+
+	$valid_mime_transforms = webp_uploads_get_upload_image_mime_transforms();
+
+	// Return early for non image or invalid mime types.
+	if ( ! isset( $valid_mime_transforms[ $mime_type ] ) ) {
+		return $quality;
+	}
```

Comment:
This should not be relevant here. We don't really care if the MIME type should be transformed or not, this filter is simply to update the WebP image quality.

Reply:
Make sense! I have updated the code to reflect this ^ Please have a look once again.